scheduler/plugin/plugin.go

```diff
@@ -57,6 +74,14 @@ func NewFromConfig(ctx context.Context, config PluginConfig) (framework.Plugin,
 		return nil, err
 	}
 
+	var guestExports exports
+	if guestExports, err = detectExports(guestModule.ExportedFunctions()); err != nil {
+		_ = runtime.Close(ctx)
+		return nil, err
+	} else if guestExports == 0 {
```

Comment:
Shouldn't we close runtime here?

Reply:
yep

Reply:
there was actually another, too. I refactored the code to make leaks less easy.